### ISSUE TITLE
async has become a reserved keyword in Python 3.7

### ISSUE THREAD

**Issue by noahfx** [FrancescoCeruti/linux-show-player#126]
Linux Show Player fails during its building due to `async` has become a reserved word
in Python 3.7 
``` Setting up linux-show-player (0.5-1) ...
    File "/usr/lib/python3/dist-packages/lisp/core/decorators.py", line 26
      def async(target):
              ^
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/core/signal.py", line 31
      from lisp.core.decorators import async
                                           ^
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/cues/cue.py", line 24
      from lisp.core.decorators import async
                                           ^
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/cues/media_cue.py", line 25
      from lisp.core.decorators import async
                                           ^
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/modules/action_cues/command_cue.py", line 25
      from lisp.core.decorators import async
                                           ^
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/modules/action_cues/volume_control.py", line 28
      from lisp.core.decorators import async
                                           ^https://github.com/FrancescoCeruti/linux-show-player/pull/127
  SyntaxError: invalid syntax
  
    File "/usr/lib/python3/dist-packages/lisp/modules/remote/controller.py", line 26
      from lisp.core.decorators import async
                                           ^
  SyntaxError: invalid syntax 
```

I  have submitted a PR with the fix https://github.com/FrancescoCeruti/linux-show-player/pull/127

Best Regads

**Comment by FrancescoCeruti** [comment]
Yes, I was aware of the problem, thanks :-)
I'll look into the pull-request as soon as possible, I will release a 0.5.2 with the fix included.

Thanks again

**Comment by FrancescoCeruti** [comment]
I've released the new [0.5.1-py3.7"](https://github.com/FrancescoCeruti/linux-show-player/releases/tag/v0.5.1-py3.7) (yes I've changed my mind on the name).

Best regards :-)
